### PR TITLE
[full-ci] Missing finedesign in Shares Panel #2

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7340

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingIntGroups</strong> https://drone.owncloud.com/owncloud/web/27330/28/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroups-shareWithGroups_feature-L257.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by tbsbdr — APPROVED** [review]
LGTM, thanks!

**Comment by JanAckermann** [comment]
While doing investigations e2e tests seems to be broken by ods bump
https://drone.owncloud.com/link/owncloud/web/tree/refs/pull/7412/head

File: tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js [hunk 1]
```diff
@@ -297,10 +297,14 @@ module.exports = {
...
-      selector: '.files-collaborators-collaborator-additional-info'
+      selector:
+        '//div[contains(@class, "share-access-details-drop")]//dt[contains(text(),"Additional info")]/following-sibling::dd[1]',
+      locateStrategy: 'xpath'
     },
     collaboratorShareType: {
-      selector: '.files-collaborators-collaborator-share-type'
+      selector:
+        '//div[contains(@class, "share-access-details-drop")]//dt[contains(text(),"Type")]/following-sibling::dd[1]',
+      locateStrategy: 'xpath'
```
Comment (JanAckermann) [inline tests/acceptance/pageObjects/FilesPageElement/SharingDialog/collaboratorsDialog.js:307]: image

**Review by kulmann — APPROVED** [review]
Nice! 💪 🎨

**Pull request merged**